platform/source/minimal-printf/README.md

```diff
@@ -10,12 +10,12 @@ Supports:
 * %d: signed integer [h, hh, (none), l, ll, z, j, t].
 * %i: signed integer [h, hh, (none), l, ll, z, j, t].
 * %u: unsigned integer [h, hh, (none), l, ll, z, j, t].
-* %x: unsigned integer [h, hh, (none), l, ll, z, j, t], printed as hexadecimal number (e.g., ff).
-* %X: unsigned integer [h, hh, (none), l, ll, z, j, t], printed as hexadecimal number (e.g., FF).
-* %f: floating point (disabled by default).
-* %F: floating point (disabled by default, treated as %f).
-* %g: floating point (disabled by default, treated as %f).
-* %G: floating point (disabled by default, treated as %f).
+* %x: unsigned integer [h, hh, (none), l, ll, z, j, t], printed as hexadecimal number (e.g., ff). Width size (up to 16) and prepending zero modifiers are supported.
+* %X: unsigned integer [h, hh, (none), l, ll, z, j, t], printed as hexadecimal number (e.g., FF). Width size (up to 16) and prepending zero modifiers are supported.
```

Comment:
The proposed implementation does not support width modifier as expected so it should be at least mentioned that the width modifier is rounded down for odd numbers. But I don't think we should accept this limitation as it will create confusion.

Reply:
@evedon Ok than I will try to find a different solution

Reply:
@evedon done. Everything should work as it is supposed to.